### DISCUSSION TITLE
KB5029253 and KB5029263

### DISCUSSION THREAD

**Discussion by SouthernHeftruck** [valinet/ExplorerPatcher#1951] [General]
There are a lot of problems with Windows 11 22H2's KB5029263 in combination with Explorer Patcher.
But are there also problems with KB50292**53** on Windows 11 **21**H2?  I can't find issues or discussions. There are a lot of computers not yet updated to 22H2.

**Comment by pyrates999** [comment]
I think only the current release of windows 11 and 10, which is 22h2, is supported by EP.

**Reply by SouthernHeftruck** [reply]
No, EP works very well on 21H2 and probably better.... 

For some reason EP works better (no problems yet) on my not windows 11 compatible MBR desktop but running updated win11 22H2 than on my new Win11 22H2 laptops with the same latest update.

**Reply by pyrates999** [reply]
Try it and find out.

**Reply by SouthernHeftruck** [reply]
Testcomputer: HP Desktop 2009, (incompatible) : Runs EP on Windows 11 21H2	KB5029253	22000.2295 without any problems AFAICS. 
Like I said, better than EP on Win 11 22H2 on a new laptop with KB5029263.
Both with latest version of EP: 56.3

**Reply by pyrates999** [reply]
I don't think that computer was compatible with windows 11 in the first place considering it's from 2009, and that you had to bypass the compatibility check to install it.  So not a valid test.
